devtools/project-core-extension-codestarts/src/main/resources/codestarts/quarkus/extension-codestarts/reactive-messaging-codestart/base/src/main/resources/application.tpl.qute.yml

```diff
@@ -0,0 +1,163 @@
+{#each input.selected-extensions-ga}
+  {#if it.equals("io.quarkus:quarkus-smallrye-reactive-messaging-kafka")}
+    {#if count == 1 }
+
+kafka:
+  bootstrap:
+    servers: localhost:9092
+
+mp:
+  messaging:
+    outgoing:
+      source-out:
+        connector: smallrye-kafka
+        value:
+          serializer: org.apache.kafka.common.serialization.StringSerializer
```

Comment:
We can remove Serde altogether as it'll be detected automatically

Reply:
fixed